src/interaction/iodevices.jl

```diff
@@ -138,8 +138,17 @@ module Keyboard
     )
 
     """
-    """
+        Keyboard.Action
+    
+    Enumerates all key states/actions in accordance with the GLFW spec.
 
+    $(INSTANCES)
+    """
+    @enum Action begin
+        release = 0
+        press   = 1
+        repeat  = 2
+    end
```

Comment:
In the last pr I tried to differentiate "is being pushed", "is held", "is being released" and "is up". I ended up with "press" and "release" but "down" and "up" are probably more common. Should I change this to down/up? Or maybe add them as aliases?